Maquetado/scss/details.scss

```diff
@@ -39,35 +52,40 @@
 
 .container-book {
   display: flex;
+  flex-direction: column;
 }
 
 .book {
   background-color: $white;
-  display: flex;
+  display: grid;
```

Comment:
You don't need use grid here because the card says: "Que el layout de la pagina sea generado usando CSS Grid" (`.list`) and "Usar grid areas para reorganizar los elementos"